chmod: add a+rX example (#6457)
tldr-pages/tldr: pages/common/chmod.md

- Change permissions recursively giving [g]roup and [o]thers the ability to [w]rite:

`chmod -R g+w,o+w {{directory}}`

> - Recursively give [a]ll users [r]ead permissions to all contents in a directory. Makes subdirectories e[X]ecutable but not files:

Reviewer: What do you think about this?
```suggestion
- Recursively give [a]ll users [r]ead and e[X]ecute permissions to a directory and its sub-directories, but not to the files it contains:
```

Author: It's better, but makes it sound like that only directories get read access, when files also get read access. Is this more clear?
```suggestion
- Recursively give [a]ll users [r]ead permissions to files and e[X]ecute permissions to sub-directories within a directory:
```

Reviewer: Perfect, I like it.

> `chmod -R a+rX {{directory}}`

Reviewer: ```suggestion
`chmod -R a+rX {{directory}}`

```
This should fix the error from `tldr-bot`.
